### DISCUSSION TITLE
Could I limit media publishing to a hand-chosen set of accounts?

### DISCUSSION THREAD

**Discussion by dogweather** [mediacms-io/mediacms#900] [General]
Hi, Everyone. MediaCMS looks amazing. Would it work for my scenario—a platform for a particular set of journalists—with two user roles:

`Everyone`:      They can comment, create favorite lists, share existing posts & download media.
`Journalists`:   All of the above **plus** create new public media posts.

In a nutshell, new media files can only be published by `Journalists`. `Everyone` accounts can't publish new videos / photos. Their UGC (user generated content) is limited to commenting and lists.

It looks like the [schools, education](https://github.com/mediacms-io/mediacms#example-cases) scenario could suit my needs.

Thanks in advance.

**Comment by mgogoulos** [comment]
Yes, that can work with very modifications. 

There is a field for logged in users that now shows as `Advanced user` on the user profile - you could easily change that to show as something else (as `Journalist `in this case). So that way you can mark accounts as Journalists. 

Then you edit cms/settings.py and add

`CAN_ADD_MEDIA = "advancedUser"`

so that only users with that setting can add medid. There are other 1-2 changes that you can perform to make it look better (eg when a non Journalist visit the upload form, they see a text describing why they can't add
